src/sql.js

```diff
@@ -4,7 +4,7 @@ const convertToEsri = require('./geometry/convert-to-esri')
 const convertFromEsri = require('./geometry/convert-from-esri')
 const sql = require('alasql')
 const geohash = require('ngeohash')
-const centroid = require('@turf/centroid')
+const centroid = require('@turf/centroid').default
```

Comment:
I am a bit confused. Does it mean getting a function named `default`, or the `export default` function?

Reply:
Also, there is no test for this.

Reply:
There are no tests for the file in general.  It will have to be a TODO to add tests; for now this is breaking functionality.

Reply:
See turf: https://github.com/Turfjs/turf/blob/master/packages/turf-centroid/test.js#L8

I guess they use approach because of typescript? https://github.com/Turfjs/turf/blob/master/packages/turf-centroid/index.ts#L35